src/common/meta/src/error.rs

```diff
@@ -278,6 +279,17 @@ pub enum Error {
 
     #[snafu(display("Retry later"))]
     RetryLater { source: BoxedError },
+
+    #[snafu(display(
+        "Failed to encode a wal options to json string, wal_options: {:?}",
+        wal_options
+    ))]
+    EncodeWalOptionsToJson {
+        wal_options: WalOptions,
+        #[snafu(source)]
+        error: serde_json::Error,
+        location: Location,
+    },
```

Comment:
```suggestion
    EncodeWalOptionsToJson {
        wal_options: WalOptions,
        source: serde_json::Error,
        location: Location,
    },
```

Reply:
If without `#[snafu(source)]`, the compiler reports an error: `serde_json::Error` does not implement debug_fmt ...

Reply:
Oh, I see